ambry-tools/src/main/java/com.github.ambry/clustermap/HelixVcrPopulateTool.java

```diff
@@ -140,7 +140,11 @@ static void createCluster(String destZkString, String destClusterName) {
     // set PersistBestPossibleAssignment
     ConfigAccessor configAccessor = new ConfigAccessor(destZkClient);
     ClusterConfig clusterConfig = configAccessor.getClusterConfig(destClusterName);
+    // if offline instances >= 4, helix enters maintenance mode.
     clusterConfig.setPersistBestPossibleAssignment(true);
+    clusterConfig.setMaxOfflineInstancesAllowed(4);
```

Comment:
Could we make the min/max values configurable?  the defaults seem okay but would be good to add optional command line args to override them.